ui/app/components/clients/page/counts.ts

```diff
@@ -127,9 +119,9 @@ export default class ClientsCountsPageComponent extends Component<Args> {
     // show banner if startTime returned from activity log (response) is after the queried startTime
     const { activity, config } = this.args;
     const activityStartDateObject = parseAPITimestamp(activity.startTime) as Date;
-    const queryStartDateObject = parseAPITimestamp(this.startTimestampISO) as Date;
+    const queryStartDateObject = parseAPITimestamp(this.args.startTimestamp) as Date;
     const isEnterprise =
-      this.startTimestampISO === config.billingStartTimestamp?.toISOString() && this.version.isEnterprise;
+      this.args.startTimestamp === config.billingStartTimestamp?.toISOString() && this.version.isEnterprise;
     const message = isEnterprise ? 'Your license start date is' : 'You requested data from';
```

Comment:
Now that the UI is defaulting to the API start/end times and not setting anything under the hood, I feel like we can update this to just be `You requested data from`. I think this means we could also remove any uses of the activity `config` model. What do you think?

Reply:
Does that language work if the user didn't set a timestamp, and it's just the default from what the API is returning? 

Unfortunately the config is still used in the template for various empty state messages, so removing it here won't eliminate our need for it

Reply:
Okay! That makes sense. I couldn't remember if this messaging was also used for default empty state management.